gitbook-docs/apps_api.md

```diff
@@ -0,0 +1,121 @@
+# Apps API
+
+When a Signal K server provides webapps (rather than just data) it SHOULD implement the Apps API. This provides a standard way to list installed webapps, 
+search for new webapps, and install or remove them. 
+
+This allows different server implementations to share webapps, which gives webapps wider markets, and reduces 'brand' dependency. Basically any 
+Signal K webapp should work on any Signal K server. This API standardises the management of those apps.
+
+To be compatible with any Signal K server Signal K webapps are required to be HTML5 webapps, which rely only on static files served from a web server, 
+and a connection to a Signal K server which supplies json data via the Signal K REST and websocket connections.
+
+## API root
+
+The Aps API root is `signalk/v1/apps/`
+
+## List
+
+`signalk/v1/apps/list` lists the currently install webapps. It is typically used to create app menus, and manage installed app lists.
+It returns a json array in the form:
+```
+[{
+//mandatory
+	"name": "Kip",
+	"description": "Really cool instrument app!!",
+	"location": "../@mxtommy/kip/package/public/index.html",
```

Comment:
@rob42 when implementing a client I'd prefer an absolute path from the root of the server. @sbender9 your thoughts from Wilhelm SK perspective?

Reply:
done